test/test_proxy_tensor.py

```diff
@@ -1301,6 +1302,7 @@ def f(a, b, c, d, e):
     xfail('unbind', ''),  # aten.unbind.int - couldn't find symbolic meta function/decomposition
 }
 symbolic_tensor_segfaults = {
+    skip('nn.functional.batch_norm')  # Segfault??
```

Comment:
hmm.. debug before land?

Reply:
I think this is just exposing an underlying bug between functionalization and symbolic shapes. @bdhirsh is looking at it.

Reply:
oh no... what I saw wasn't a segfault 😛. I guess we can see if my patch fixes this though.

Reply:
It's sometimes a segfault, sometimes the error you reported.

Reply:
fwiw, it doesn't look like this has to do with functionalization. I patched my fix, and what I noticed from some poking around is:

(1) The segfault happens [here](https://github.com/pytorch/pytorch/blob/317eeb81c3e7ab21ca4359819c4a89122ce574f5/torch/csrc/jit/python/init.cpp#L1390). At some point, we must be calling `.get_pyobj()` on a symint that isn't actually a `PySymIntNodeImpl` (or just that's in an invalid state).
(2) It happens before any functionalization code runs